grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -219,7 +220,7 @@ class ProductAvailabilityService {
      *  2. Picklist items from pending orders (outbound returns) with origin being provided location.
      *     (IMPORTANT: Outbound returns can have picked items with RECALLED lots and bins with HOLD_STOCK activity)
      * */
-    def getQuantityPickedByProductAndLocation(Location location, Product product) {
+    List<QuantityPickedByLocationAndProductDto> getQuantityPickedByProductAndLocation(Location location, Product product) {
```

Comment:
Same with this one. Lose the Dto and call it something like AllocatedItem or AllocationItem. Or StockAllocationItem could also work since it follows the same pattern as descriptive pattenr as StockMovementItem. But I'm open to other suggestions.

Reply:
Decided to use AllocatedItem. Should live in the same package as AvailableItem although that is an inner class in StockMovementItem. 

org.pih.warehouse.api.AllocatedItem